src/prefect/settings/models/cloud.py

```diff
@@ -39,6 +39,11 @@ class CloudSettings(PrefectBaseSettings):
         description="API URL for Prefect Cloud. Used for authentication with Prefect Cloud.",
     )
 
+    disable_orchestration_telemetry: bool = Field(
+        default=False,
+        description="Whether or not to disable orchestration telemetry.",
+    )
```

Comment:
I have a slight preference to use positive voicing for settings like this, but I'm happy for pushback on this.
```suggestion
    enable_orchestration_telemetry: bool = Field(
        default=True,
        description="Whether or not to disable orchestration telemetry.",
    )
```

Reply:
No pushback, positive voicing it is.